app/src/main/java/com/besome/sketch/editor/PropertyActivity.java

```diff
@@ -161,27 +163,35 @@ public void onBackPressed() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.property);
+
+		content = findViewById(R.id.content);
+		layout_ads = findViewById(R.id.layout_ads);
+		scroll_view = findViewById(R.id.scroll_view);
+		property_group_list = findViewById(R.id.property_group_list);
```

Comment:
Please fix formatting here.

Reply:
So now what should I fix here? Or briefly, should I revert the changes? Or should I close this pull request? Very strange reasons.

Reply:
I just want you to fix the formatting.